security.md

```diff
@@ -149,6 +140,16 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
+### Hash your customer's passwords with a proper password hashing function
+
+
+If you can afford it, use a [third party authentication service](#customer-users-management) to handle password storage, password management and password recovery. 

```

Comment:
@cottsak Regarding your first point. Hashing passwords right is only 1 task out of 4 or 5 that you need to get right in order to have a proper login experience. think about MFA, about password policies, about proper forgot-my-password. These are today a de-facto standard, and the developers would need to code that too. Take a look at [Okta Platform](http://developer.okta.com/documentation/) for example. They provide both white labelling (meaning you can use them as an authentication service API), or you can use them as a user interface to provide your customers. Their pricing is active monthly users, so it could work pretty well even for small startups.

For the second point, I didn't think to put the hashing password section at all, just because I really believe developers have no business writing that code themselves. So I prefer to cut down on subjects, but have each subject explicitly defined in terms of pros cons. Although I agree we will have to refactor it to be more wiki like (vs one long document).